src/install/installMultiple.ts

```diff
@@ -403,7 +410,8 @@ async function install (
         engines: pkg.engines,
         cpu: pkg.cpu,
         os: pkg.os,
-      }
+      },
+      cacheByNodeVersion: ctx.force || pkgResponse.body.sideEffectsCache[NODE_MAJOR],
```

Comment:
actually, it can be simplified even more.

This object has a property called `path`. `path` is used only once, when `centralLocation` is created. So central location can be calculated here, instead of in `resolvePeers.ts`. In that case this object will need to contain one property instead of two. So only `centralLocation` instead of `path` and `cacheByNodeVersion`

Reply:
And how do we mark `isBuilt`? Do we really need to do this? It seems more complicated to me.

Reply:
`isBuilt` can be added to this object as well

Reply:
but maybe lets wait with this. I am still thinking about our discussion in package-requester. If the cache can depent on dependency set, this code will have to be changed more and a different place

Reply:
So we are replacing one field for another? I started doing what you asked for and I had to add way more lines of code than those I removed... Should I go ahead? Really?